ArmJoint turret
ArmJoint shoulder
ArmJoint elbow1
ArmJoint elbow2
ArmJoint tilt
ArmJoint pan
ArmJoint wrist
ArmJoint grip
